File: yushengjun.py
class ysj:
    def __init__(self,theme):
        print('_'*39)
        print(theme)
        self.ind = 1
        self.ash = '|   '
        self.dawn = '\033[0m'
        self.dawn = '\033[90m'
        self.dawn1 = '\033[91m'
        self.dawn2 = '\033[92m'
        self.dawn4 = '\033[94m'
    
    def adl(self):
        self.ind += 1
    def pur(self):
        print('_' * 39)
    def voidline(self):
        print(f'{self.dawn2}{self.ash*self.ind}')
    def printline(self,string):
        print(f'{self.dawn2}{self.ash*self.ind}{self.dawn}{string}')
    def inputline(self,string):
        return input(f'{self.dawn2}{self.ash*self.ind}{self.dawn}{string}{self.dawn4}_')
    def callline(self,string,object):
        print(f'{self.dawn2}{self.ash*self.ind}{string}{self.dawn4}{object}')
'''
pr = ysj("test")
pr.voidline()
pr.adl()
pr.printline("别炸了！")
'''

File: yushengjun.c
#include <stdio.h>

#define dawn "\033[0m"
#define dawn0 "\033[90m"
#define dawn1 "\033[91m"
#define dawn2 "\033[92m"
#define dawn3 "\033[93m"
#define dawn4 "\033[94m"
#define dawn5 "\033[95m"

int ind = 1;
const char* ash = "|   ";

void urln();
void vdln();
void prln(const char* string);
char* inln(const char* string);
void clln(const char* string, const char* object);

int main() {
    /* code here.*/
    return 0;
}

void urln() {
    printf("_");
}

void vdln() {
    printf("%s", ash);
}

void prln(const char* string) {
    printf("%s%s%s%s", YELLOW, ash, RESET, string);
}

char* inln(const char* string) {
    printf("%s%s%s_ ", YELLOW, ash, RESET);
    return fgets(stdin, 100, stdin);
}

void clln(const char* string, const char* object) {
    printf("%s%s%s%s%s%s", YELLOW, ash, string, PURPLE, object, RESET);
}
